src/olympia/amo/utils.py

```diff
@@ -642,7 +641,21 @@ def resize_image(source, destination, size=None, *, format='png', quality=80):
             im.load()
     original_size = im.size
     if size:
-        im = processors.scale_and_crop(im.convert('RGBA'), size)
+        try:
+            target_width, target_height = [int(v) for v in size]
+            source_width, source_height = [float(v) for v in im.size]
+            scale = min(target_width / source_width, target_height / source_height)
+            if scale <= 1:
+                im = im.convert('RGBA')
+                im = im.resize(
+                    (
+                        int(round(source_width * scale)),
+                        int(round(source_height * scale)),
+                    ),
+                    resample=Image.LANCZOS,
+                )
+        except ValueError as value_error:
+            print(value_error)
```

Comment:
We don't want to leave `print()`s around. In fact, we probably don't want to `try/except` at all here: if a `ValueError` is raised from the code above something is wrong in the logic and should be handled there rather than hide the error.